pkg/drivers/kic/oci/errors.go

```diff
@@ -26,3 +26,9 @@ var ErrWindowsContainers = FailFastError(errors.New("docker container type is wi
 
 // ErrCPUCountLimit is thrown when docker daemon doesn't have enough CPUs for the requested container
 var ErrCPUCountLimit = FailFastError(errors.New("not enough CPUs is available for container"))
+
+// ErrExitedAfterCreate is thrown when container is created without error but it exists and it's status is not running.
+var ErrExitedAfterCreate = errors.New("container status is not running after creation")
```

Comment:
I find this sentence difficult to read. Perhaps something like:

`container exited unexpectedly`

Reply:
done